rep-2000.rst

```diff
@@ -679,11 +679,11 @@ Dependency Requirements:
 +-------------+----------------+---------------+----------------+----------------+---------------------+
 | PCL         |     1.10.0     |     N/A       |     N/A        |      1.9.1     |       1.10.0        |
 +-------------+----------------+---------------+----------------+----------------+---------------------+
-|                                **RMW DDS Milddleware Providers**                                     |
+|                                **RMW DDS Middleware Providers**                                     |
 +-------------+----------------+---------------+----------------+----------------+---------------------+
 | Connext DDS |                              5.3.1              |               N/A                    |
 +-------------+----------------+---------------+----------------+----------------+---------------------+
-| Cyclone DDS |                                 0.5.1 (Eusebius)                                       |
+| Cyclone DDS |                                 0.6.0                                                  |
```

Comment:
How about using 0.6.x here since patch releases might happen over time and Foxy is tracking the 0.6.x branch?

Reply:
sure! made that change and you know it will change to 0.7.x quite soon